### PR TITLE
Updates AmCharts (changes)

### PR DESCRIPTION
**Pull request by mlzummo**
Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
floating column has been refactored into bar ... candle needs to have the same done with it as well.

File: src/map/Choropleth.js [hunk 1]
```diff
@@ -28,6 +28,7 @@
     Choropleth.prototype.publish("useClonedPalette", false, "boolean", "Enable or disable using a cloned palette",null,{tags:["Intermediate","Shared"]});
 
     Choropleth.prototype.data = function (_) {
+        console.log(data);
```
Comment (GordonSmith) [inline src/map/Choropleth.js]: remove console logging

File: src/chart/XYAxis.js [hunk 2]
```diff
@@ -208,7 +208,7 @@
             return d.map(function (item) {
                 return this.formatValue(item);
             }, this);
-        } 
+        }
```
Comment (GordonSmith) [inline src/chart/XYAxis.js]: remove whitespace change

File: src/amchart/CommonXY.js [hunk 3]
```diff
@@ -99,7 +99,7 @@
         if (this.marginTop()) { this._chart.marginTop = this.marginTop(); }
         if (this.marginBottom()) { this._chart.marginBottom = this.marginBottom(); }
 
-        this._chart.dataDateFormat = this.dataDateFormat();
+        //this._chart.dataDateFormat = this.dataDateFormat();
```
Comment (GordonSmith) [inline src/amchart/CommonXY.js]: remove commented code.

File: src/amchart/CommonXY.js [hunk 4]
```diff
@@ -57,7 +57,7 @@
     CommonXY.prototype.publish("marginTop", 20, "number", "Margin (Top)",null,{tags:["Intermediate"]});
     CommonXY.prototype.publish("marginBottom", 50, "number", "Margin (Bottom)",null,{tags:["Intermediate"]});
 
-    CommonXY.prototype.publish("dataDateFormat", null, "string", "",null,{tags:["Private"]});
+    //CommonXY.prototype.publish("dataDateFormat", null, "string", "",null,{tags:["Private"]});
```
Comment (GordonSmith) [inline src/amchart/CommonXY.js]: remove commented code.

File: src/amchart/CommonSerial.js [hunk 5]
```diff
@@ -242,17 +505,14 @@
         var context = this;
         var initObj = {
             type: "serial",
-            chartScrollbar: {}
+            chartScrollbar: {},
```
Comment (GordonSmith) [inline src/amchart/CommonSerial.js]: remove trailing comma

**Comment by mlzummo** [comment]
@GordonSmith suggested changes made please re-review

File: src/amchart/CommonRadar.js [hunk 6]
```diff
@@ -47,7 +47,7 @@
 
     CommonRadar.prototype.publish("startDuration", 0.3, "number", "Start Duration (sec)",null,{tags:["Private"]});
 
-    CommonRadar.prototype.publish("dataDateFormat", null, "string", "Date Format String",null,{tags:["Private"]});
+    //CommonRadar.prototype.publish("dataDateFormat", null, "string", "Date Format String",null,{tags:["Private"]});
```
Comment (GordonSmith) [inline src/amchart/CommonRadar.js]: Remove commented code

File: src/amchart/CommonRadar.js [hunk 7]
```diff
@@ -75,7 +75,7 @@
     CommonRadar.prototype.updateChartOptions = function() {
         var context = this;
 
-        if (this.dataDateFormat()) { this._chart.dataDateFormat = this.dataDateFormat(); }
+        //if (this.dataDateFormat()) { this._chart.dataDateFormat = this.dataDateFormat(); }
```
Comment (GordonSmith) [inline src/amchart/CommonRadar.js]: Remove commented code

**Comment by mlzummo** [comment]
@GordonSmith made suggested changes ...

**Pull request merged**